### PR TITLE
Implement shippo API-definitions

### PR DESCRIPTION
**Pull request by nanobash**
## Summary

[Preview](https://preview.redoc.ly/rebilly-users/shippo-api-defs/tag/Websites/operation/GetWebsiteCollection#!c=200&path=settings/shippo&t=response)

- [x] Implement website settings.shippo object
- [x] Implement Shippo credentials

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Websites/WebsiteSettings.yaml [hunk 1]
```diff
@@ -26,3 +26,33 @@ properties:
...
+        type: string
+        description: Shippo weight.
+      weightUnit:
+        type: string
+        enum:
+          - g
+          - oz
+          - lb
+          - kg
+        description: Shippo weightUnit.
```
Comment (justnero) [inline openapi/components/schemas/Websites/WebsiteSettings.yaml:52]: This is not an enum, but a template that evaluates to the enum.
```suggestion
      weightUnit:
        type: string
        description: Shippo weight unit. Supports mustache templates within the invoice context. Must evaluate to a valid Shippo weight unit: `g`, `oz`, `lb` or `kg`.
```

File: openapi/components/schemas/Websites/WebsiteSettings.yaml [hunk 2]
```diff
@@ -26,3 +26,33 @@ properties:
...
+        description: Specifies whether Shippo is enabled.
+      credentialHash:
+        type: string
+        description: Shippo credential hash.
+      filter:
+        type: string
+        description: Shippo filter.
+      weight:
+        type: string
+        description: Shippo weight.
```
Comment (justnero) [inline openapi/components/schemas/Websites/WebsiteSettings.yaml:44]:
```suggestion
      weight:
        type: string
        description: Shippo weight. Supports mustache templates within the invoice context. Must evaluate to a positive decimal.

```

File: openapi/components/schemas/Websites/WebsiteSettings.yaml [hunk 3]
```diff
@@ -26,3 +26,33 @@ properties:
...
+        type: string
+        enum:
+          - g
+          - oz
+          - lb
+          - kg
+        description: Shippo weightUnit.
+      manufactureCountry:
+        type: string
+        description: Shippo manufacture country.
```
Comment (justnero) [inline openapi/components/schemas/Websites/WebsiteSettings.yaml:55]:
```suggestion
      manufactureCountry:
        type: string
        description: Shippo manufacture country. Supports mustache templates within the invoice context. Must evaluate to a valid ISO 3166 alpha-2 country code.

```

File: openapi/components/schemas/Websites/WebsiteSettings.yaml [hunk 4]
```diff
@@ -26,3 +26,33 @@ properties:
...
+      sku:
+        type: string
+        description: Shippo accounting code.
```
Comment (justnero) [inline openapi/components/schemas/Websites/WebsiteSettings.yaml:58]:
```suggestion
      sku:
        type: string
        description: Shippo accounting code. Supports mustache templates within the invoice context.
```

File: openapi/components/schemas/Websites/WebsiteSettings.yaml (hunk 1 above)
Comment (nanobash) [reply openapi/components/schemas/Websites/WebsiteSettings.yaml]: Updated, thanks

**Comment by nanobash** [comment]
[The RFC is closed.](https://github.com/Rebilly/docs-product/pull/1312)

**Pull request closed without merging**